pwiz/analysis/spectrum_processing/SpectrumList_IonMobility.cpp

```diff
@@ -98,6 +99,14 @@ SpectrumList_IonMobility::SpectrumList_IonMobility(const msdata::SpectrumListPtr
                 units_ = IonMobilityUnits::compensation_V;
             else if (!scan.userParam("drift time").empty()) // Oldest known mzML drift time style
                 units_ = IonMobilityUnits::drift_time_msec;
+            for (int b = 0; b < spectrum->binaryDataArrayPtrs.size(); b++)

```

Comment:
If you don't need b as a counter/output variable, you can do range-based for (the modern C++ equivalent of C#'s foreach):
`for (const auto& binaryArrayPtr : spectrum->binaryDataArrayPtrs)`

Reply:
Nice, will do.